src/core/system_impl.h

```diff
@@ -50,6 +50,52 @@ class SystemImpl : public Sender {
         Stabilized,
     };
 
+    // Enumeration representing the available modes for the Arudpilot rover autopilot.
+    enum class APRoverMode {
```

Comment:
Usually we try the convention to lowercase acronyms like so: `ApRoverMode`, however, I assume you copied them from somewhere, so then maybe it's better leave em as is.